monitor.py

```diff
@@ -428,6 +428,44 @@ def get_failed_monitor_testcases(spy_link,job_type):
     else:
         return "Failed to get response from e2e-test directory url"
 
+def get_testcase_frequency(spylinks, zone, tc_name = None):
+    """
+    To get the testcases failing with its frequency
+
+    Args:
+        spylinks (list): list of builds which needs to be checked.
+        zone (list): List of the zones/leases that need to checked.
+        tc_name (list): list of testcase name.
+
+    Returns:
+        dict: Dict with testcase as key and its frequency as value
+
+    """
+    frequency = {}
+    for spylink in spylinks:
+        job_type,_ = job_classifier(spylink)
+        lease,_ = get_quota_and_nightly(spylink)
+        if zone is not None and lease not in zone :
+            continue
+        cluster_status=cluster_deploy_status(spylink)
+        if cluster_status == 'SUCCESS' and "4.15" not in spylink:
```

Comment:
why does this function not support 4.15 builds?

Reply:
Removed it here and get_failed_testcases() as well.